configs/config/pretrain/vision_transformer/swav/swav_deit_b_2nodes.yaml

```diff
@@ -3,7 +3,6 @@ config:
   HOOKS:
     TENSORBOARD_SETUP:
       USE_TENSORBOARD: True
-      EXPERIMENT_LOG_DIR: "/checkpoint/ito/vision_transformer/1gpu_test"
```

Comment:
we should rebase to master. This change should be essentially reverted after the rebase.